mod/deckorate_search/set/self/search/open_search.rb

```diff
@@ -92,4 +92,14 @@ def os_type_filter
 
     yield[:filter] = { term: { type_id: type_param.card_id } }
   end
+
+  # constructs the context filtering clause for the suggest_query
+  # in case of multiple contexts we are favoring other contexts than source
+  def suggest_contexts
+    return filter_type_ids unless filter_type_ids.length() > 1
```

Comment:
can you add the code suggested in slack and then make a new PR to main?

(I want to see if that triggers the code climate analysis.  I don't like merging in code that hasn't been through that).